.github/workflows/ensure-backported.yml

```diff
@@ -0,0 +1,29 @@
+
+name: Ensure changes are migrated to Vue3
+
+on:
+  pull_request:
+    types: [opened, ready_for_review, reopened, synchronize, labeled]
```

Comment:
Maybe it should be on merged?

Reply:
Nobody cares after it's merged, so should be done beforehand.

But I'm not sure if that's a good flow:
- open PR
- get red CI
- trigger bot
- rerun CI again

Reply:
As soon as the bot adds the label the `labeled` trigger will cause the CI to be green - no manual interaction needed.